facade/publicendpoint.go

```diff
@@ -145,31 +160,44 @@ func (f *Facade) RemovePublicEndpointPort(ctx datastore.Context, serviceid, endp
 	if err != nil {
 		err = fmt.Errorf("Could not find service %s: %s", serviceid, err)
 		glog.Error(err)
-		return err
+		return alog.Error(err)
 	}
+	alog = alog.Entity(svc)
 
 	err = svc.RemovePort(endpointName, portAddr)
 	if err != nil {
 		err = fmt.Errorf("Error removing port %s from service (%s): %v", portAddr, svc.Name, err)
 		glog.Error(err)
-		return err
+		return alog.Error(err)
 	}
 
 	glog.V(2).Infof("Removed port public endpoint %s from service %s", portAddr, svc.Name)
 
 	if err = f.UpdateService(ctx, *svc); err != nil {
 		glog.Error(err)
-		return err
+		return alog.Error(err)
 	}
 
 	glog.V(2).Infof("Service (%s) updated", svc.Name)
+	alog.Succeeded()
 	return nil
 }
 
 // Enable/Disable a port public endpoint.
 func (f *Facade) EnablePublicEndpointPort(ctx datastore.Context, serviceid, endpointName, portAddr string, isEnabled bool) error {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.EnablePublicEndpointPort"))
-
+	var auditMsg string
+	if isEnabled {
+		auditMsg = "Enabling Public Endpoint Port"
+	} else {
+		auditMsg = "Disabling Public Endpoing Port"
+	}
+	alog := f.auditLogger.Message(ctx, auditMsg).Action(audit.Update).ID(serviceid).
+		WithFields(logrus.Fields{
+			"endpointname": endpointName,
+			"portaddr": portAddr,
+			"isenabled": isEnabled,
```

Comment:
Since, we are showing appropriate message, I think `isenabled` field is not necessary.